crherman7/rechunk
# feat(www): eslint + prettier config

## Scope

## Implementation

#### Eslit updates

* move `eslint` config  to shared package
* added `prettier` for `www`
* apply lint rules for `www` app

## Screenshots

<details>
  <summary>example for Separator component</summary>

  ### Before
  <p align="center">
    <img src="https://github.com/user-attachments/assets/32666abb-b484-4a63-b70b-b1a9989dae4a" width="750">
  </p>
  
  ### After
  <p align="center">
    <img src="https://github.com/user-attachments/assets/bf41b22e-ce16-4846-a4d1-e47941603569" width="750">
  </p>
</details>



## How to Test

1. open any `ts/tsx` file in `www`
2. observe that basic plugins like `simple-import-sort` are used

## Emoji Guide

**For reviewers: Emojis can be added to comments to call out blocking versus non-blocking feedback.**

E.g: Praise, minor suggestions, or clarifying questions that don’t block merging the PR.

> 🟢 Nice refactor!

> 🟡 Why was the default value removed?

E.g: Blocking feedback must be addressed before merging.

> 🔴 This change will break something important

| | | |
| --- | --- | --- |
| Blocking | 🔴 ❌ 🚨 | RED |
| Non-blocking | 🟡 💡 🤔 💭 | Yellow, thinking, etc |
| Praise | 🟢 💚 😍 👍 🙌 | Green, hearts, positive emojis, etc |
